src/librustdoc/html/static/js/search.js

```diff
@@ -2334,11 +2334,16 @@ ${item.displayPath}<span class="${type}">${name}</span>\
     }
 
     function makeTabHeader(tabNb, text, nbElems) {
+        // https://blog.horizon-eda.org/misc/2020/02/19/ui.html
```

Comment:
Very nice read. Could you add a small explanation nonetheless in case the website disappears? Something like `\u{2007} is a unicode character the same size as a digit to keep alignment locked`.